src/WorkflowCore/Services/BackgroundTasks/RunnablePoller.cs

```diff
@@ -51,27 +49,14 @@ private async void PollRunnables(object target)
         {
             try
             {
-                if (await _lockProvider.AcquireLock("poll runnables", new CancellationToken()))
+                _logger.LogInformation("Polling for runnable workflows");
```

Comment:
We don't need here to create a generic lock for the hole polling process. Each single item will be protected by the "wf:item" lock. So multiple processes can safely enqueue concurrently items in the queue